evalml/tests/model_understanding_tests/prediction_explanations_tests/test_force_plots.py

```diff
@@ -0,0 +1,203 @@
+from itertools import product
+from unittest.mock import patch
+
+import pandas as pd
+import pytest
+import shap
+
+from evalml.demos import load_breast_cancer, load_wine
+from evalml.model_understanding.force_plots import force_plot, graph_force_plot
+from evalml.pipelines import (
+    BinaryClassificationPipeline,
+    MulticlassClassificationPipeline,
+    RegressionPipeline,
+)
+
+
+def test_exceptions():
+    with pytest.raises(
+        TypeError,
+        match="rows_to_explain should be provided as a list of row index integers!",
+    ):
+        force_plot(None, None, None, None)
+    with pytest.raises(
+        TypeError, match="rows_to_explain should only contain integers!"
+    ):
+        force_plot(
+            pipeline=None,
+            rows_to_explain=["this", "party's", "over"],
+            training_data=None,
+            y=None,
+        )
+
+
+@patch("evalml.model_understanding.force_plots.jupyter_check")
+@patch("evalml.model_understanding.force_plots.initjs")
+@pytest.mark.parametrize(
+    "rows_to_explain, just_data", product([[0], [0, 1, 2, 3, 4]], [False, True])
+)
+def test_force_plot_binary(
```

Comment:
Do you think it's worth adding a test to make sure the force plot displays the aggregate contribution of features that have been OHE/text features? I vote yes just because I think it's important to make sure we maintain that behavior since the reports also aggregate the OHE/text features.

Reply:
I do....I just need to figure out what that looks like, haha.